drivers/spi/spi_cc23x0.c

```diff
@@ -339,16 +360,22 @@ static int spi_cc23x0_transceive(const struct device *dev,
 	block_cfg_rx.dest_addr_adj = DMA_ADDR_ADJ_INCREMENT;
 	block_cfg_rx.block_size = SPI_CC23_DFS * data->tx_len_left;
 
-	ret = dma_config(cfg->dma_dev, cfg->dma_channel_tx, &dma_cfg_tx);
+	ret = pm_device_runtime_get(cfg->dma_dev);
```

Comment:
@bjarki-andreasen runtime get will do policy lock automatically right? or is it pending a merge?